code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -799,13 +804,48 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	update_icon()
 	add_overlay(icon_alert)
+/obj/item/pda/proc/receive_ping(message)
```

Comment:
add new line before

Reply:
Don't know why it was resolved but not fixed, but this needs a new line